pdf-email/functions/pdf.js

```diff
@@ -0,0 +1,127 @@
+/**
+ *
+ *  This Function shows you how to create and send pdf on the fly using Twilio Sendgrid.
+ *  The main idea is that files created on the fly within a Twilio Function are always stored under the /tmp/ directory
+ *  So after the pdf making, we grant the relevant permissions to the file so that we can access the contents and send it via email as normal
+ *  IMPORTANT: Use this function if you want to perform an action that does not require permanent storage as the contents of /tmp/ are removed - preferably single function invocation logic
+ *
+ *  Pre-requisites
+ *  - You need to have a Twilio Sendgrid API KEY
+ *  - You need to include the following npm modules: pdfmake, fs, pdf-to-base64, request, node-cmd
```

Comment:
These should all be in the `package.json` in the `dependencies` field. That will install them automatically for people. The easiest way for you to add them is to run `npm install pdfmake ...` inside the `pdf-email` folder

Reply:
ok cool will change that, thought these would be added from the functions console